hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestHiveSyncTool.java

```diff
@@ -84,31 +87,32 @@ public void testSchemaConvertArray() throws IOException {
     MessageType schema = Types.buildMessage().optionalGroup().as(OriginalType.LIST).repeatedGroup()
```

Comment:
minor. do we need to fix line 79(old) / 82 (new) for java doc. it links to parquet list. guess we are moving to Avro with this diff.